lisa/schema.py

```diff
@@ -434,7 +436,7 @@ class DiskControllerType(str, Enum):
 @dataclass()
 class DiskOptionSettings(FeatureSettings):
     type: str = constants.FEATURE_DISK
-    disk_type: Optional[
+    os_disk_type: Optional[
```

Comment:
Does os disk support all types?

Reply:
No, OS disk does not support Ultra Disk.